pkgs/shells/fish/plugins/wakatime-fish.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, wakatime, buildFishPlugin, fetchFromGitHub }:
+
+buildFishPlugin rec {
+  pname = "wakatime-fish";
+  version = "0.0.3";
+
+  src = fetchFromGitHub {
+    owner = "ik11235";
+    repo = "wakatime.fish";
+    rev = "v${version}";
+    hash = "sha256-t0b8jvkNU7agF0A8YkwQ57qGGqcYJF7l9eNr12j2ZQ0=";
+  };
+
+  buildInputs = [
+    wakatime
```

Comment:
Looking at other plugins (including zsh), it seems that we should not add the external command to `buildInputs`.